microsoft/testsuites/gpu/gpusuite.py

```diff
@@ -247,14 +247,28 @@ def verify_gpu_cuda_with_pytorch(
         _install_driver(node, log_path, log)
         _check_driver_installed(node, log)
 
-        _install_cudnn(node)
+        # pytorch and CUDA needs 4GB space to download and install
+        free_partition = node.find_partition_with_freespace(5)
+        node.execute(
+            f"cd {free_partition}; mkdir lisawork; chmod -R 777 lisawork;",
+            shell=True,
+            sudo=True,
+        )
+        work_path = free_partition + "/lisawork"
+        log.debug(f"Current work dir is {work_path}")
 
-        gpu = node.features[Gpu]
+        _install_cudnn(node, log, work_path)
 
         pip = node.tools[Pip]
+        pip._install()
```

Comment:
1. don't call private method out if its class.
2. install will be called automatically, it doesn't need to be called explicitly.

Reply:
removed